### ISSUE TITLE
pip package missing cryptography dependency

### ISSUE THREAD

**Issue by koliyo** [StackStorm/st2#4408]
##### SUMMARY

Installed using pip package

```
pip3 install -U st2client
```

Got error for missing dependency package

```
$ st2
Traceback (most recent call last):
  File "/usr/local/bin/st2", line 7, in <module>
    from st2client.shell import main
  File "/usr/local/lib/python3.7/site-packages/st2client/shell.py", line 27, in <module>
    from cryptography.utils import CryptographyDeprecationWarning
ImportError: cannot import name 'CryptographyDeprecationWarning' from 'cryptography.utils' (/usr/local/lib/python3.7/site-packages/cryptography/utils.py)
```

Manually install dep
```
pip3 install -U cryptography
```

Now it works

```
$ st2 --version
st2 2.9.1, on Python 3.7.0
```

##### ISSUE TYPE
Pick one below and delete the rest:
 - Bug Report

##### STACKSTORM VERSION
Paste the output of `st2 --version`:

st2 2.9.1, on Python 3.7.0

##### STEPS TO REPRODUCE

```yaml

```

**Comment by Kami** [comment]
Thanks for reporting this.

I can confirm this issue.

We made some changes in the past which should prevent such issues from appearing, but it looks like those weren't enough.

I will push a fix shortly and I will also look into adding test cases which will allow us to automatically catch such regression in the future.

**Comment by LindsayHill** [comment]
Fixed by #4409
